consensus/checks.go

```diff
@@ -190,7 +190,7 @@ func (consensus *Consensus) onViewChangeSanityCheck(recvMsg *FBFTMessage) bool {
 }
 
 func (consensus *Consensus) onNewViewSanityCheck(recvMsg *FBFTMessage) bool {
-	if recvMsg.ViewID <= consensus.viewID {
+	if recvMsg.ViewID < consensus.viewID {
```

Comment:
why is it ok to allow a new view for the same view id as the last successful view id?

Reply:
This is more of a temporary workaround to solve the view change stuck issue. We do see it in mainnet and other scenarios that this check got view change stuck. Likely the consensus.viewID is updated incorrectly somewhere else. Before we have the total view change revamp. I think it's ok to make this change to make view change more robust.

Reply:
sure. I will address this later. thanks for explanation.